### PR TITLE
update: debugLevel defaults to 0 in production

### PR DESCRIPTION
**Pull request by Rennzie**
Closes https://github.com/GeoTIFF/georaster-layer-for-leaflet/issues/57

### CODE REVIEW

File: georaster-layer-for-leaflet.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const PROJ4_SUPPORTED_PROJECTIONS = new Set([3857, 4269]);
 const MAX_NORTHING = 1000;
 const MAX_EASTING = 1000;
 const ORIGIN = [0, 0];
+const IS_PRODUCTION = process.env.NODE_ENV === "production";
```
Comment (DanielJDufour) [inline georaster-layer-for-leaflet.js:12]: In order to play it safe, could we change this to the following?
```js
const IS_PRODUCTION = typeof process === 'object' && typeof process.env === 'object' && process.env.NODE_ENV === "production";
```

I'm wondering if anyone might be trying to directly load `georaster-layer-for-leaflet.js` in the browser where process won't be defined.

**Review by DanielJDufour — CHANGES_REQUESTED** [review]
Thanks for pushing this forward.  I left a small comment.

File: georaster-layer-for-leaflet.js (hunk 1 above)
Comment (Rennzie) [reply georaster-layer-for-leaflet.js:12]: Ah yes good point, I hadn't considered that. Is there a specific reason you are checking the `typeof` at each step or could we just to: 

```javascript
 const IS_PRODUCTION  = process?.env?.NODE_ENV
```

which would return `false` if any of those where undefined or null.

Comment (Rennzie) [reply georaster-layer-for-leaflet.js:12]: Another option could be simply to set the default level as `0` and make the user set it to `1` when they want to see the output.

**Comment by Rennzie** [comment]
Hey @DanielJDufour I've made the changes you requested. 

Let me know what the think re my reply to your review comment, particularly your thoughts on simplifying this and setting default to 0 and letting the user define when they want to see the logs.

**Comment by DanielJDufour** [comment]
Hi, @Rennzie .  Sorry for the delay.  I think simplifying it and just defaulting the logging to zero is a good idea.  I don't have strong opinions either way.  Feel free to update this PR and I'll get that merged in.  Thank you.

**Comment by Rennzie** [comment]
No worries. 

Ok cool that sound good. Simpler is better so will update pr later today.

**Comment by Rennzie** [comment]
@DanielJDufour this is ready to go 🚀

**Pull request merged**